assets/panel/debugger.properties

```diff
@@ -405,6 +405,12 @@ breakpointMenuItem.removeCondition.label=Remove breakpoint condition
 breakpointMenuItem.removeCondition.accesskey=c
 breakpointMenuItem.editCondition.label=Edit breakpoint condition
 breakpointMenuItem.editCondition.accesskey=n
+breakpointMenuItem.disableAllAtLine.label=Disable breakpoints on line
+breakpointMenuItem.disableAllAtLine.accesskey=d
```

Comment:
D, uppercase

Reply:
`D` is already taken

Reply:
Usage: accesskeys are case insensitive, so you have a duplicate anyway.

"Disable breakpoints on line" will still highlight the first uppercase `D`, because it falls back to the other case if the original one is missing.

Reply:
> Usage: accesskeys are case insensitive, so you have a duplicate anyway.

That, unless DevTools do accesskeys in a different way compared to any other software I know.